### DISCUSSION TITLE
Transmission Web Control无法访问

### DISCUSSION THREAD

**Discussion by WindowsXp-Beta** [natfrp/wiki#273] [使用问题]
我给家里NAS的Transmission Web Control添加了一条TCP隧道，导向了9091端口。frpc服务成功启动，但是按照给出的`域名:端口`访问，浏览器里显示`ERR_SSL_PROTOCOL_ERROR`。DSM桌面的5001端口已经穿透成功了，不知道问题出在了什么地方？

**Comment by fakeboboliu** [comment]
为对应隧道打开 自动https 并重启隧道再试
![图片](https://user-images.githubusercontent.com/7552030/215725005-193743df-88e1-4f27-83ab-c1d6b8fb42c2.png)

**Reply by WindowsXp-Beta** [reply]
感谢解答，尝试了之后发现无法用域名访问（一样的报错），但可以用ip访问。
